app/buck2_file_watcher/src/watchman/core.rs

```diff
@@ -208,6 +208,8 @@ async fn with_timeout<R>(
 }
 
 async fn write_to_manifold(buf: &[u8], name: &str) -> Option<String> {
+    // FIXME(jadel): thread configuration through
+    #[expect(deprecated)]
     let manifold = ManifoldClient::new().await.ok()?;
```

Comment:
nit, less lines and no 'deprecated' tagging. Internally 'deprecated' doesn't mean much ¯\_(ツ)_/¯ 

```suggestion
    Self::new_with_config(None).await
```